.travis.yml

```diff
@@ -9,6 +9,8 @@ before_install:
   - wget 'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-linux-x86_64.tar.bz2'
   - tar xf phantomjs-1.9.7-linux-x86_64.tar.bz2
   - export PATH=$PATH:phantomjs-1.9.7-linux-x86_64/bin/
+  - psql -c "CREATE USER accounts WITH SUPERUSER PASSWORD 'accounts'" -U postgres
```

Comment:
How does tutor-server not need to set this? https://github.com/openstax/tutor-server/blob/master/.travis.yml

Anything we can do to make the approach in Account mirror the existing approach in tutor-server is a good thing.